### ISSUE TITLE
Map View++ rendering of linestrings

### ISSUE THREAD

**Issue by LorenzBuehmann** [ad-freiburg/qlever-ui#50]
Hi QLever team,

not an issue but more a question: It looks like the Map view++ widget isn't capable of rendering linestrings, is that assumption correct?

I tried to visualize the telecommunication net, the expected output is something like this in Overpass turbo: 

![image](https://user-images.githubusercontent.com/976029/188615752-b71bac0a-88c0-472e-be8b-0acb6f3597e5.png)

using the Overpass query

```
/*
This query looks for nodes, ways and relations 
with the given key/value combination.
Choose your region and hit the Run button above!
*/
[out:json][timeout:50];
// gather results
(
  // query part for: “communication=line”
  node["communication"="line"]({{bbox}});
  way["communication"="line"]({{bbox}});
  relation["communication"="line"]({{bbox}});
);
// print results
out body;
>;
out skel qt;
```

In QLever (OSM Planet) we can gather the same data with

``` sparql
PREFIX geo: <http://www.opengis.net/ont/geosparql#>
PREFIX osmkey: <https://www.openstreetmap.org/wiki/Key:>

SELECT ?s ?lit WHERE {
    ?s osmkey:communication "line" . 
    ?s geo:hasGeometry ?lit
} 
```
but the Map view++ only shows
![image](https://user-images.githubusercontent.com/976029/188616391-41760b8f-73d3-4bd4-9e90-fddd9545bd38.png)

Debugging by having a look at one of the longer ways in OSM browser view, e.g. [this one](https://www.openstreetmap.org/way/459190538#map=4/46.56/-23.03)

![image](https://user-images.githubusercontent.com/976029/188616851-ae642870-d19b-4894-b542-6ee9dfa870e6.png)

I can see the point of the linestring where the bend is here in Map view++
![image](https://user-images.githubusercontent.com/976029/188617088-77bbd0ec-36c1-4632-9573-d24b573aa157.png)

but the rest of the line is missing.

Long story short, is this the expected behavior? It's a bit confusing when those line strings are missing I think.

Minor issue: the "normal" map view doesn't show any results at all on the map. Is it broken?

Kind regards,
Lorenz

**Comment by patrickbr** [comment]
Sorry for the late answer - we fixed this a couple of months ago, your [example query](https://qlever.cs.uni-freiburg.de/mapui-petri/?query=PREFIX%20geo%3A%20%3Chttp%3A%2F%2Fwww.opengis.net%2Font%2Fgeosparql%23%3E%20PREFIX%20osmkey%3A%20%3Chttps%3A%2F%2Fwww.openstreetmap.org%2Fwiki%2FKey%3A%3E%20SELECT%20%3Fs%20%3Flit%20WHERE%20%7B%20%3Fs%20osmkey%3Acommunication%20%22line%22%20.%20%3Fs%20geo%3AhasGeometry%20%3Flit%20%7D&backend=https%3A%2F%2Fqlever.cs.uni-freiburg.de%2Fapi%2Fosm-planet) now shows the full lines when the map is rendered in "Objects" mode (map button on the left).

**Comment by LorenzBuehmann** [comment]
Works perfectly, even in Heatmap mode. 
Thanks.
